docs/source/optimizations/cq/cq_compile_qss.ipynb

```diff
@@ -21,26 +21,57 @@
     "</a>"
```

Comment:
<p>Update install instructions to match other <code>qss</code> optimization notebooks </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>